src/OnboardingSPA/pages/Steps/GetStarted/Welcome/Sidebar/LearnMore/contents.js

```diff
@@ -55,16 +54,17 @@ const getContents = ( brandName, expertsLink, techSupportLink, fullServiceCreati
 						translations( 'website' )
 					),
 					description: sprintf(
-						/* translators: 1: Website 2: Brand */
+						/* translators: 1: Website 2: Brand 3. Site */
 						__(
-							`A WordPress %s hosted by %s has tons of exclusive, easy
-						and powerful solutions to help you get farther,
-						faster with your WordPress -- we put our expertise, partnerships
-						and solutions to work for you.`,
+							`A WordPress %s hosted by %s has tons of unique and proven
+						solutions to help you get farther,
+						faster with your WordPress. We put our expertise, partnerships
+						and solutions to work on your %s.`,
 							'wp-module-onboarding'
 						),
 						translations( 'website' ),
-						brandName
+						brandName,
+						translations('website')
```

Comment:
Hey @0aveRyan , just wanted to confirm the third substitute here - as per the translators mentioned on line 57, it is `site`, but here it is mentioned as `website`. Could you please confirm what is to be used as substitute - `site` or `website`?

Reply:
@Yashita101 let's go with `site`, thanks!